<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_o"/>
    <addaction name="action_6"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(E)</string>
    </property>
    <addaction name="action_ASCII"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="fileopen"/>
   <addaction name="filenew"/>
   <addaction name="change_2"/>
   <addaction name="filesave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>38</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>524287</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>打开照片</string>
     </property>
    </widget>
    <widget class="QFontComboBox" name="fontComboBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="change">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>转化成ascii</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>输入ASCII码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="changeToGray">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>转化成灰度图</string>
     </property>
    </widget>
    <widget class="QPushButton" name="changeFilter">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>均值滤波</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="birghtness">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-255</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>310</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>亮度：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>360</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>对比度：</string>
     </property>
    </widget>
    <widget class="QSlider" name="contrast">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>380</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="brightButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>410</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>亮度生成</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gaussButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>450</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>高斯模糊</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sobelButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>490</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Sobel处理</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cannyButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>530</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Canny处理</string>
     </property>
    </widget>
    <widget class="QPushButton" name="overButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>570</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>复合运算</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>数字签名</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveimgButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>保存图片</string>
     </property>
    </widget>
    <widget class="QPushButton" name="erButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>二值化</string>
     </property>
    </widget>
    <widget class="QSlider" name="nSlider">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>81</width>
       <height>19</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>95</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="gammaButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>伽马变换</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>81</width>
       <height>19</height>
      </rect>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="openVideoButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>270</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>打开视频</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="filenew">
   <property name="icon">
    <iconset>
     <normaloff>../2020-2021-1 QT上机任务5-应用程序主窗口/images/filenew.png</normaloff>../2020-2021-1 QT上机任务5-应用程序主窗口/images/filenew.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="fileopen">
   <property name="icon">
    <iconset>
     <normaloff>../2020-2021-1 QT上机任务5-应用程序主窗口/images/fileopen.png</normaloff>../2020-2021-1 QT上机任务5-应用程序主窗口/images/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="change_2">
   <property name="icon">
    <iconset>
     <normaloff>../2020-2021-1 QT上机任务5-应用程序主窗口/images/change.png</normaloff>../2020-2021-1 QT上机任务5-应用程序主窗口/images/change.png</iconset>
   </property>
   <property name="text">
    <string>转化</string>
   </property>
  </action>
  <action name="filesave">
   <property name="icon">
    <iconset>
     <normaloff>../2020-2021-1 QT上机任务5-应用程序主窗口/images/filesave.png</normaloff>../2020-2021-1 QT上机任务5-应用程序主窗口/images/filesave.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_o">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="action_ASCII">
   <property name="text">
    <string>转成ASCII码</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>转成灰度图</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>均值滤波</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>边缘检测</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>边缘检测复合</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>保存(S)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
